proto/iotics/api/meta.proto

```diff
@@ -168,3 +143,58 @@ message SparqlUpdateResponse {
   // SPARQL update response headers
   Headers headers = 1;
 }
+
+// FKGQueryRequest describes a SPARQL query.
+message FKGQueryRequest {
+
+  // SPARQL query request payload.
+  message Payload {
+
+    // The desired result content type. Note that choosing an invalid result type for the type of query will result in
+    // an error status reported in the response. (See SparqlResultType for valid content-query type combinations.)
+    SparqlResultType resultContentType = 1;
```

Comment:
can you document the behaviour if resultContentType isn't specified

Reply:
Sure - I'll add the default

Reply:
24d01b6 (basically it'll be the default enum value since the field cannot be "empty")